src/earn/EarnPoolInfoScreen.tsx

```diff
@@ -107,14 +110,162 @@ function TokenIcons({
+        <Text numberOfLines={1} style={styles.depositAndEarningsCardLabelText}>
+          {earnItem.label}
+        </Text>
+      </View>
+      <View style={styles.flexShrink}>
+        <Text style={styles.depositAndEarningsCardValueText}>
+          {t('earnFlow.poolInfoScreen.lineItemAmountDisplay', {
+            localCurrencySymbol,
+            localCurrencyAmount: formatValueToDisplay(amountInLocalCurrency),
+            cryptoAmount: formatValueToDisplay(new BigNumber(earnItem.amount)),
+            cryptoSymbol: tokenInfo?.symbol,
+          })}
+        </Text>
+      </View>
+    </View>
+  )
+}
+
+function DepositAndEarningsCard({ earnPosition }: { earnPosition: EarnPosition }) {
+  const { t } = useTranslation()
+  const { balance } = earnPosition
+  const { earningItems, depositTokenId, cantSeparateCompoundedInterest } = earnPosition.dataProps
+  const tokenInfo = useTokenInfo(depositTokenId)
+  const localCurrencySymbol = useSelector(getLocalCurrencySymbol)
+  const localCurrencyExchangeRate = useSelector(usdToLocalCurrencyRateSelector)
+
+  // Deposit items used to calculate the total balance and total deposited
+  const depositBalanceInUsd = tokenInfo?.priceUsd?.multipliedBy(balance)
+  const depositBalanceInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+    depositBalanceInUsd ?? 0
+  )
+
+  // Earning items used to calculate the total balance and total deposited
+  const earningItemsTokenIds = earningItems.map((item) => item.tokenId)
+  const earningItemsTokenInfo = useTokensInfo(earningItemsTokenIds)
+
+  const totalBalanceInLocalCurrency = depositBalanceInLocalCurrency.plus(
+    earningItems.reduce((acc, item) => {
+      if (item.includedInPoolBalance) return acc
+      const tokenInfo = earningItemsTokenInfo.find((token) => token?.tokenId === item.tokenId)
+      const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(item.amount)
+      const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+        amountInUsd ?? 0
+      )
+      return acc.plus(amountInLocalCurrency ?? 0)
+    }, new BigNumber(0))
+  )
+
+  const totalDepositBalanceInLocalCurrency = depositBalanceInLocalCurrency.minus(
+    earningItems.reduce((acc, item) => {
+      if (!item.includedInPoolBalance) return acc
+      const tokenInfo = earningItemsTokenInfo.find((token) => token?.tokenId === item.tokenId)
+      const amountInUsd = tokenInfo?.priceUsd?.multipliedBy(item.amount)
+      const amountInLocalCurrency = new BigNumber(localCurrencyExchangeRate ?? 0).multipliedBy(
+        amountInUsd ?? 0
+      )
+      return acc.plus(amountInLocalCurrency ?? 0)
+    }, new BigNumber(0))
+  )
+
+  const totalDepositBalanceInCrypto = new BigNumber(balance).minus(
+    earningItems.reduce((acc, item) => {
+      if (!item.includedInPoolBalance) return acc
+      const itemToken = earningItemsTokenInfo.find((token) => token?.tokenId === item.tokenId)
+      return acc.plus(
```

Comment:
can we add test coverage for these?

Reply:
These should all have coverage now from the tests added in e00254f5cf5b796566f5a435127bb8bbbb765c8c.